cmake/CMakeLists.txt

```diff
@@ -4,39 +4,58 @@ cmake_minimum_required(VERSION 2.8)
 # Project
 project(protobuf C CXX)
 
+# CMake policies
+cmake_policy(SET CMP0022 NEW)
+
 # Options
+option(protobuf_VERBOSE "Enable for verbose output" OFF)
```

Comment:
This is not a good practice to use short variable names for the project settings.
Usually VERBOSE option is only needed to debug a specific project.
If this project is included in the build process of a larger project, and it is quite possible with Git and CMake, then the settings of the current project will be coordinated with the policies of other project? ..
I would also like to rename the option ZLIB in protobuf_WITH_ZLIB. What do you think about it?